test/support/idx/factories/remediations.ts

```diff
@@ -521,3 +524,15 @@ export const SelectAuthenticatorUnlockAccountRemediationFactory = IdxRemediation
     AuthenticatorValueFactory.build()
   ]
 });
+
+export const ResendAuthenticatorFactory = IdxRemediationFactory.params({
```

Comment:
Looks like this factory is not used in tests

Reply:
Correct, I ended up not using this factory, but there is no harm in adding it